content/methods/future-methods/ht-ratio-estimation.jsonnet

```diff
@@ -8,7 +8,10 @@
     Theme: 'Sample Design & Estimation',
     Author: 'ONS',
     'Programming language': 'Python/PySpark',
-    'Code available?': 'Yes',
+    "Release": "Not Released Yet",
```

Comment:
As discussed Access Type and Programming Language should be changed as per the mock up / JIRA ticket